src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -249,10 +264,21 @@ export const canShow = async (
 		hasRequiredConsents(),
 	]);
 
+	await maybeWipeUserData(apiKey as string, brazeUuid);
```

Comment:
personal preference would be to have `as string` defined on the line with `const apiKey`

Reply:
Ah the issue here is that where it's defined `apiKey` isn't necessarily a string (it might not be set) but we'll return early if it's not truthy. So we'll only get to this line if it's a string, but TypeScript for some reason doesn't get that, hence the casting.